packages/core/src/addresses/redux/actions/doDeleteDefaultShippingAddress.js

```diff
@@ -0,0 +1,53 @@
+import * as actionTypes from '../actionTypes.js';
+import { getDefaultAddress } from '../reducer.js';
+
+/**
+ * Responsible for deleting the users default Shipping address.
+ *
+ * @param deleteDefaultShippingAddress - Delete default Shipping address client.
```

Comment:
Please, check the jsdoc warnings of all files as there are some warnings related to missing parameters

Reply:
Done 👍🏻